packaging/rpm-ostree.spec.in

```diff
@@ -116,6 +116,7 @@ python autofiles.py > files \
   '%{_sysconfdir}/dbus-1/system.d/*' \
   '%{_prefix}/lib/systemd/system/*' \
   '%{_libexecdir}/rpm-ostree*' \
+  '%{_datadir}/dbus-1/interfaces/org.projectatomic.rpmostree1.xml' \
```

Comment:
(Since this failed, can you either drop this hunk, or just copy the current fedora dist-git master here?)